cmd/snapctl/flags.go

```diff
@@ -43,6 +43,16 @@ var (
 		Name:  "running",
 		Usage: "Shows running plugins",
 	}
+	flPassword = cli.BoolFlag{
+		Name:  "password, p",
+		Usage: "Password for REST API authentication",
+	}
+	flPasswordPath = cli.StringFlag{
+		Name:   "Password, P",
```

Comment:
We should come up with a better ENV name and command line flag that doesn't include PASSWORD or PASS in the name. A username and password flag on the command line should be different from an auth or credential file being passed in. Ideally, I would say snapctl should have the option of a config file that could hold this information and other config related options.